### PR TITLE
Move verdi data to click

### PR DESCRIPTION
**Pull request by yakutovicha**
1) Move verdi data to click
2) Split subcommands into separate files
3) Add tests

### CODE REVIEW

File: aiida/cmdline/commands/data/bands.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+from aiida.cmdline.commands.data.export import _export
+from aiida.cmdline.params.options.multivalue import MultipleValueOption
+from aiida.cmdline.commands import verdi_data
+from aiida.cmdline.params import arguments
+from aiida.cmdline.params import options
+from aiida.cmdline.utils import echo
+from aiida.orm.data.array.bands import BandsData
+from aiida.common.utils import Prettifier
+
+# if not is_dbenv_loaded():
```
Comment (DropD) [inline aiida/cmdline/commands/data/bands.py]: remove unused comments, especially code like this, which would potentially slow down all of verdi.

File: aiida/cmdline/commands/data/bands.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+#     load_dbenv()
+
+def show_xmgrace(exec_name, list_bands):
+    """
+    Plugin for show the bands with the XMGrace plotting software.
```
Comment (DropD) [inline aiida/cmdline/commands/data/bands.py]: "Plugin for showing ... "

File: aiida/cmdline/commands/data/bands.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            sys.exit(1)
+        else:
+            raise
+
+
+@verdi_data.group('bands')
+@click.pass_context
+def bands(ctx):
+    """
+    Manipulation of the bands
```
Comment (DropD) [inline aiida/cmdline/commands/data/bands.py]: """Manipulate BandsData objects."""

Comment (DropD) [reply aiida/cmdline/commands/data/bands.py]: or 'bands objects'

File: aiida/cmdline/commands/data/bands.py [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+    """
+    pass
+
+
+@bands.command('show')
+@arguments.NODES()
+@click.option('-f', '--format', 'format',
```
Comment (DropD) [inline aiida/cmdline/commands/data/bands.py]: Define the format option only once.

**Comment by szoupanos** [comment]
@yakutovicha since you did the final check of the commands, it would be nice if you could write which flags of which commands are not tested in order to also ask the help of @merkys before the final merge. I suppose that it is deposit but I am not sure about the status of the export which is shared by many data types.

**Comment by merkys** [comment]
@szoupanos I don't have an AiiDA instance set up for tests right now, and setting up one would take quite some time, I guess. I got an impression that this PR is mostly code restructuring, therefore, I don't expect it to break something. Of course, it's better to check, but as said, this would take some time.

File: aiida/cmdline/commands/data/bands.py (hunk 1 above)
Comment (yakutovicha) [reply aiida/cmdline/commands/data/bands.py]: done

File: aiida/cmdline/commands/data/bands.py (hunk 2 above)
Comment (yakutovicha) [reply aiida/cmdline/commands/data/bands.py]: done

File: aiida/cmdline/commands/data/bands.py (hunk 3 above)
Comment (yakutovicha) [reply aiida/cmdline/commands/data/bands.py]: done

File: aiida/cmdline/commands/data/bands.py (hunk 4 above)
Comment (yakutovicha) [reply aiida/cmdline/commands/data/bands.py]: done

**Comment by szoupanos** [comment]
@merkys E.g. I was/we were unable to find any tests about code deposition. Yes, theoretically, we just changed the "interface" - the way we call your data related functions but since there was a lot of work to be done in a few days, I am sure that there will be problems for the untested parts.

It is not urgent, I suppose, but it is good to write from our side what has been tested & checked (@yakutovicha will do the final list) and you can at least give your opinion/comments.

This will help the next person who will have to work on this.

**Comment by yakutovicha** [comment]
There were some python formatting problems, now all checks passed.
@DropD please review the changes again.

**Comment by merkys** [comment]
@szoupanos you are absolutely right. I will check as soon as I am able to allocate some time for it. It will also be a great opportunity to refresh the knowledge about the development of AiiDA.

**Pull request merged**